app/src/main/java/com/sopt/now/SignUpActivity.kt

```diff
@@ -0,0 +1,83 @@
+package com.sopt.now
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.material.snackbar.Snackbar
+import com.sopt.now.databinding.ActivitySignupBinding
+
+class SignUpActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySignupBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignupBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+
+        binding.btnSignup2.setOnClickListener {
+            val signid = binding.etIdSignup.text
+            val signpw = binding.etPwSignup.text
+            val signname = binding.etNicknameSignup.text
+            val signmbti = binding.etMbtiSignup.text
+
+            if (signid.length < 6 || signid.length > 10) {
```

Comment:
```suggestion
            if(signid.length !in 6..10)
```

이런 식으로 kotlin 문법 in을 사용해서 바꿀 수도 있답니다!

Reply:
넵! 이 리뷰 보고 compose 과제에서는 그렇게 적용해 보았습니다. 감사합니다.

Reply:
수치 부분들도 상수화 해두면 좋아요!

Reply:
넵! 감사합니다